data/src/main/java/me/androidbox/data/local/dao/EventDao.kt

```diff
@@ -1,25 +1,21 @@
 package me.androidbox.data.local.dao
 
 import androidx.room.*
-import kotlinx.coroutines.flow.Flow
 import me.androidbox.data.local.DatabaseConstant
 import me.androidbox.data.local.entity.EventEntity
 
 @Dao
 interface EventDao {
-    @Query("SELECT * FROM ${DatabaseConstant.EVENT_TABLE} WHERE `from` >= :startTimeStamp AND `from` <= :endTimeStamp")
-    fun getEventsFromTimeStamp(startTimeStamp: Long, endTimeStamp: Long): Flow<List<EventEntity>>
+    @Query("SELECT * FROM ${DatabaseConstant.EVENT_TABLE} WHERE `startDateTime` >= :startTimeStamp AND `startDateTime` <= :endTimeStamp")
+    suspend fun getEventsFromTimeStamp(startTimeStamp: Long, endTimeStamp: Long): List<EventEntity>
```

Comment:
Why did you stopped using Flows?

Reply:
@MKiperszmid I was having trouble with it. I don't think I was using it correctly. And I didn't want to spend too much time trying to fix it. However, I will come back to this later and maybe change it back to flows again.

Reply:
Yeah I'd definitely make this observable in some form. If you now use `suspend` this will make the calling code quite different, so I'd recommend to go through this rather now. Not sure what went wrong for you, but you don't really need to do anything with this flow other than observing it in the UI